content/pages/features.html

```diff
@@ -347,21 +372,22 @@ <h3 class="ui small header">
   </div>
 </section>
 
+<!-- Second CTA -->
 <section>
   <div class="ui padded container">
     <div class="ui vertically padded centered grid">
       <div class="center aligned column">
         <div class="ui very padded raised inverted secondary large segment">
           <p>
-            Start automating your documentation with Read the Docs.
+            Automate your documentation deployments with Read the Docs.
           </p>
           <p>
             <button
                class="ui large inverted button"
-               onclick="jQuery('#about-modal').modal('show');"
+               onclick="jQuery('#features-modal').modal('show');"
                type="button">
               <i class="fad fa-play icon"></i>
-              Get started now
+              Get started free
```

Comment:
This should at least be:

```suggestion
              Get started for free
```

Reply:
Though I guess I normally try to avoid mentioning "free" for the negative value association/etc.

We don't really mention the trial anywhere major I think, so I lean towards something like this to obfuscate mentioning "free".

```suggestion
              Start your 1 month trial
```

Either one is fine though really.